### PR TITLE
Some code sample errors

### PR DESCRIPTION
**Pull request by khwilson**
Small amount of cleanup to make sure code samples run (down to about 10 non-working samples, which I think are actual bugs as opposed to silly dependency issues. :-) )

### CODE REVIEW

File: Data_Manipulation/Reshaping/reshape_panel_data_from_long_to_wide.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ pop_wide <- pivot_wider(population,
 
 Another way to do this is using `data.table`.
 
-```r
+```r?example=pivot_wider
```
Comment (grantmcdermott) [inline Data_Manipulation/Reshaping/reshape_panel_data_from_long_to_wide.md:131]: I must confess that I've never used this `lang?description` chunk syntax before. Is it a rouge specific thing?

**Review by grantmcdermott — APPROVED** [review]
LGMT!

(I am curious about the chunk syntax header I feel like this going to be a TIL thing...)

File: Data_Manipulation/Reshaping/reshape_panel_data_from_long_to_wide.md (hunk 1 above)
Comment (aeturrell) [reply Data_Manipulation/Reshaping/reshape_panel_data_from_long_to_wide.md:131]: New to me too despite being familiar with codebraid, jupytext, jupyterbook, and rmarkdown... thought I'd seen all the markdown code chunk execution tools out there!

**Comment by grantmcdermott** [comment]
Ah, I see @khwilson describes it in https://github.com/LOST-STATS/lost-stats.github.io/issues/121

I'm going to go ahead an merge then. Nice work!

**Comment by khwilson** [comment]
Yep! It's rouge specific. I actually had to submit a PR to VSCode to get it to highlight correctly b/c rouge does it different from react 😆 https://github.com/microsoft/vscode-markdown-tm-grammar/pull/86

File: Data_Manipulation/Reshaping/reshape_panel_data_from_long_to_wide.md (hunk 1 above)
Comment (khwilson) [reply Data_Manipulation/Reshaping/reshape_panel_data_from_long_to_wide.md:131]: To say here, yeah, this is rouge specific. I was annoyed that it wasn't one of the other bajillion ways too

**Pull request merged**